src/javascript/lib/pdf.jsx

```diff
@@ -5,6 +5,10 @@ import Page from '../models/Page.jsx'
 
 const NO_OP = () => {}
 
+if (typeof document == "undefined") {
+  require('./domstubs.js').setStubs(global);
+}
```

Comment:
I'd prefer to put this in `pdf2md.js`, since the stubbing concerns setting up the plumbing needed for the logic to work, not the logic itself

Reply:
Agreed, makes sense.